### DISCUSSION TITLE
Running ArchiveBox Under Path Prefix

### DISCUSSION THREAD

**Discussion by hohle** [ArchiveBox/ArchiveBox#1112] [Q&A] [answered]
On my home server I partition different apps by a path prefix that `nginx` routes to the various servers running in jails on the same host. I'm trying to set up ArchiveBox to run under the `/archive` prefix, but it appears that `SCRIPT_NAME`, `FORCE_SCRIPT_NAME`, etc. don't work well with the Django development server (as best I can tell).

I'd be happy to run ArchiveBox using a WSGI host (Apache, nginx, or one of the Python app servers), but I don't know much about Django, ArchiveBox's configuration, or WSGI. Are there any examples of running ArchiveBox using a "production" web server?

**Comment by pirate** [comment, marked as answer]
This is deliberately not supported because it's a security risk, see here: https://github.com/ArchiveBox/ArchiveBox/issues/724

ArchiveBox is unique in that you're rehosting a lot of untrusted JS and user content on your domain, and so it needs stricter domain isolation (for CORS, CSRF, etc. protection) than most other self-hosted apps.

I recommend keeping archivebox on its own subdomain, and setting your `/archive` nginx handler to redirect to or iframe archivebox instead.

**Reply by hohle** [reply]
Thanks for the quick response. I suppose that's reasonable.
